queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+select * from animals where name like '%mon%';
+select * from animals where date_of_birth between '2016-01-01' and '2019-12-31';
+select * from animals where neutered = true and escape_attempts < 3;
+select date_of_birth from animals where name = 'Agumon' or name = 'Pikachu';
+select name, escape_attempts from animals where weight_kg > 10.5;
+select * from animals where neutered = true;
+select * from animals where name != 'Gabumon';
+select * from animals where weight_kg >= 10.4 and weight_kg <= 17.3;
```

Comment:
- [ ] Excellent job writing queries for the following quests👏🏻 however it's one of the project requirements you take a screenshot of the results of your queries and add them to the pull request description. This also helps to ensure that you run the queries with success. Kindly make this 👍🏻

Reply:
Thank you

Reply:
Hi @karam084 !! The screenshot of the results of your queries must be added to the pull request mean here!!